### PR TITLE
Randomize shape

### PR DESCRIPTION
**Pull request by tpummer**
Just a tiny little addition to generate some random shapes for inspiration.

### CODE REVIEW

**Comment by PyMaster22** [comment]
Pretty good addition.

**Comment by sn0wm3** [comment]
this looks pretty nice, i spent the last 30 minutes writing a more or less the same thing for myself before i saw the PR.

I think it might make more sense to have the view random shape button placed next to the buttons to try other demo shapes, though, to keep the UI consistent

**Comment by sn0wm3** [comment]
also, if you're generating a completely random shape, quadrants should be allowed to be empty; as it is your code doesn't allow for the generation of -- pieces

**Comment by tpummer** [comment]
i welcome your additions

**Comment by PyMaster22** [comment]
> also, if you're generating a completely random shape, quadrants should be allowed to be empty; as it is your code doesn't allow for the generation of `--` pieces

@sn0wm3 It could try to make an empty layer.
But I do see that @tpummer fixed it.

**Comment by tpummer** [comment]
It was already stated in the code for showing an custom typed shape 
`throw new Error("Empty layers are not allowed");`

File: index.js [hunk 1]
```diff
@@ -373,7 +375,24 @@ window.randomShape = () => {
...
+		if(randomShape === '-') {
+			randomColor = '-';
+			console.log('in');
+		}
+		layertext = layertext + randomShape + randomColor;
+	}
+	//empty layer not allowed
+	if(layertext === '--------'){
+		layertext = '';
+		y--;
```
Comment (sn0wm3) [inline index.js:392]: You made a small error here; in 392, y is no longer in scope, and 391 is unnecessary given that you're declaring the variable at the start of the loop. try replacing it with `i--`

Comment (tpummer) [reply index.js]: 👍 thx

**Comment by tobspr** [comment]
Cool, thanks!

**Pull request merged**